src/databricks/labs/ucx/source_code/files.py

```diff
@@ -150,3 +150,29 @@ def resolve_import(self, path_lookup: PathLookup, name: str) -> MaybeDependency:
 
     def __repr__(self):
         return "LocalFileResolver()"
+
+
+class SitePackageResolver(BaseDependencyResolver):
+
+    def __init__(
+        self,
+        file_loader: FileLoader,
+        site_packages_path: Path,
+        next_resolver: BaseDependencyResolver | None = None,
+    ):
+        super().__init__(next_resolver)
+        self._file_loader = file_loader
```

Comment:
@ericvergnaud : Question from the other PR. Why does the site package resolver require a file loader?